### PR TITLE
Completed react-function-components.

### PR DESCRIPTION
**Pull request by leanne-vu**
Screen Shot 2022-12-14 at 9 27 49 AM

### CODE REVIEW

File: react-function-components/src/index.jsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+function CustomButton() {
+  return <button>Click Me!</button>;
+}
+const container = document.querySelector('#root');
+const root = ReactDOM.createRoot(container);
+root.render(CustomButton());
```
Comment (Zhiwutian) [inline react-function-components/src/index.jsx:8]: I would use the react Syntax here `<CustomButton />`

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update!

**Comment by leanne-vu** [comment]
Screen Shot 2022-12-14 at 9 37 05 AM

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**